tests/Stoa.test.ts

```diff
@@ -64,7 +64,7 @@ describe ('Test of Stoa API Server', () =>
         return stoa_server.stop().then(() => { return agora_server.stop() });
     });
 
-    it ('Test of the path /block_externalized', (doneIt: () => void) =>
+    before ('Test of the path /block_externalized', (doneIt: () => void) =>
```

Comment:
This looks wrong, it's not using promises and `before` doesn't make sense here.

Reply:
All the tests that follow this are subordinate to this.
I changed it because I thought it would be a good idea to run the tests individually.
For now, I change it to the original one.